packages/cli/tests/e2e/function/Arm_TabAddFunctionBot.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @author Di Lin <dilin@microsoft.com>
+ */
+
+import path from "path";
+import "mocha";
+import * as chai from "chai";
+
+import {
+  getSubscriptionId,
+  getTestFolder,
+  getUniqueAppName,
+  cleanUp,
+  setSimpleAuthSkuNameToB1Bicep,
+  readContextMultiEnv,
+  getActivePluginsFromProjectSetting,
+  getProvisionParameterValueByKey,
+  setBotSkuNameToB1Bicep,
+} from "../commonUtils";
+import { environmentManager } from "@microsoft/teamsfx-core";
+import { CliHelper } from "../../commonlib/cliHelper";
+import { Capability, Resource, provisionParametersKey } from "../../commonlib/constants";
+import { FunctionValidator } from "../../commonlib";
+
+describe("Configuration successfully changed when with different plugins", function () {
```

Comment:
can we put these test cases into 1 ts file?

Reply:
Same as above.